.gitignore

```diff
@@ -195,6 +195,7 @@ node_modules/
 
 wwwroot
 **/Localization/**/*.po
+!test/OrchardCore.Tests/Localization/**/*.po
```

Comment:
Why?

Reply:
You must do this, or you will lose the translated documents. You can try it.

Reply:
You must do this, or you will lose the translated documents. You can try it.